### DISCUSSION TITLE
[2023-08-21] GUI Team Demo Day

### DISCUSSION THREAD

**Discussion by vitvakatu** [enso-org/enso#7621] [Weekly Demo Day]
Last week I was working on 
- #7618 
- and also fixed a minor bug https://github.com/enso-org/enso/pull/7594

https://github.com/enso-org/enso/assets/6566674/66afe9da-028e-42fe-9685-d8c3a39a68d8

**Comment by kazcw** [comment]
I've been working on context restoration (#7255). I expect to finish on Wednesday.

**Comment by MichaelMauderer** [comment]
* #7619
![Peek 2023-08-21 13-40](https://github.com/enso-org/enso/assets/1428930/32d8f066-c80d-4915-8133-6dbc1edf70b3)
![Peek 2023-08-21 13-41](https://github.com/enso-org/enso/assets/1428930/a1e5aa7e-8776-4438-9c1e-80ac1afd0292)
* #7620
* ![Peek 2023-08-21 14-32](https://github.com/enso-org/enso/assets/1428930/64018a5b-46a2-4e91-8e87-77e091c43a73)

**Comment by Frizi** [comment]
Was on vacation for most of the week.
- Currently working on implementing new design for list editor #7434, discussed an alternative design with braces present and will continue working on that.
<img width="145" alt="image" src="https://github.com/enso-org/enso/assets/919491/5becd179-88d5-4ca4-8b33-817f6d2e2682">


- Still have issues merging #7507 due to CI errors, we talked about it and plan to test builds on non-windows platforms to troubleshoot that.

**Comment by hubertp** [comment]
Relevant changes from backend:
- Notification about project rename #7613
